pom.xml

```diff
@@ -39,7 +36,7 @@
     <module>cluster</module>
     <module>cluster-api</module>
     <module>examples</module>
-    <module>utils</module>
+    <module>cluster-testlib</module>
```

Comment:
if it's a test lib you can consider an approach -> https://maven.apache.org/plugins/maven-jar-plugin/examples/create-test-jar.html and move all classes in test scope.

Reply:
@segabriel  looks like I have chosen *The preferred way*